staging/src/k8s.io/kubectl/pkg/scale/scale.go

```diff
@@ -112,6 +113,16 @@ type genericScaler struct {
 	scaleNamespacer scaleclient.ScalesGetter
 }
 
+// objectForReplicas is used for encoding scale patch
+type objectForReplicas = struct {
+	Replicas uint `json:"replicas"`
+}
+
+// objectForSpec is used for encoding scale patch
+type objectForSpec = struct {
```

Comment:
Why are you aliasing?

Reply:
I am a little confused about this comment. Here is a nested struct, so I declare `objectForSpec` and `objectForReplicas`.

Reply:
Type alias to anonymous struct:
```go
type objectForSpec = struct {
  ...
}
```
Regular struct deceration:
```go
type objectForSpec struct {
  ...
}
```

Reply:
Thanks for clarifying. done